be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -155,6 +156,10 @@ class SegmentIterator::BitmapRangeIterator {
     uint32_t _buf_pos = 0;
     uint32_t _buf_size = 0;
     bool _eof = false;
+
+public:
+    uint32_t cur_ = 0;
+    roaring::Roaring bitmap_;
```

Comment:
we can get roaring from iterator->parent, so we do not need to store it seperately